### ISSUE TITLE
Means of input and output REQ 3

### ISSUE THREAD

**Issue by JWJPaton** [w3c/apa#150]
The base requirement is that the device/service is accessible so we should be clear that a non-natural language alternative that fulfils an accessibility needs is acceptable. 

I suggest changing
"REQ 3:Support multiple input devices and methods."
to 
"REQ 3: Support multiple input devices and methods either within the natural language interface/s or via alternatives."

**Comment by jasonjgw** [comment]
If someone wants to make a graphical user interface (nonvisually, let's suppose) accessible, then it isn't a good answer to tell them to create a command line interface that offers the same functionality. It might well be useful to have the command line interface also, but it isn't solving the original problem of making the GUI accessible.
It seems to me that the situation discussed here is exactly parallel: "write an alternative interface" might be useful advice to give users flexibility, but it doesn't solve the original accessibility problem with the natural language interface. Our document is devoted to the "how to make the natural language interface accessible" problem, whereas other guidance such as WCAG can be used if it is decided to make an equivalent GUI Web-based interface as well, for example.
Are these the right distinctions to draw? Should we clarify it in the Introduction?

**Comment by jasonjgw** [comment]
I propose the following solution.

Add a statement to the Scope section which indicates that creating a parallel user interface that does not involve natural language interaction may enhance accessibility. However, it is outside the scope of this document, and addressed in other W3C guidance such as the Web content Accessibility Guidelines.
On the other hand, I think there may be aspects of multimodal interaction which should be considered within the scope of this work, if they raise specific accessibility challenges.

**Comment by RealJoshue108** [comment]
@jasonjgw can we bring this up in our upcoming discussions on NAUR. I like @JWJPaton suggestion and have merged it into main - but think we need to pick up this thread again ourselves.
